xbmc/pvr/PVRGUIActions.cpp

```diff
@@ -1482,32 +1480,43 @@ namespace PVR
     pDlgProgress->SetPercentage(10);
     pDlgProgress->Progress();
 
-    /* reset the EPG pointers */
-    const CPVRDatabasePtr database(CServiceBroker::GetPVRManager().GetTVDatabase());
```

Comment:
Is this the same database as for radio?

Reply:
Yep. There is only one for both tv and radio.